x/bank/app_test.go

```diff
@@ -22,6 +22,8 @@ import (
 	"github.com/cosmos/cosmos-sdk/x/bank/testutil"
 	"github.com/cosmos/cosmos-sdk/x/bank/types"
 	_ "github.com/cosmos/cosmos-sdk/x/consensus"
+	_ "github.com/cosmos/cosmos-sdk/x/distribution"
+	dk "github.com/cosmos/cosmos-sdk/x/distribution/keeper"
```

Comment:
```suggestion
	distrkeeper "github.com/cosmos/cosmos-sdk/x/distribution/keeper"
```

Nit, I believe we usually call it `distrkeeper`.